tree/dataframe/src/RDataFrame.cxx

```diff
@@ -629,6 +629,8 @@ RDataFrame operations currently work with this package. The subset that is curre
 - Redefine
 - Snapshot
 - Sum
+- Systematic variations: Vary and VariationsFor
+- Parallel submission of distributed graphs: RunGraphs
```

Comment:
RunGraphs and VariationsFor are a bit special as they are not RDF methods but rather free functions. Can we link to their docs from here?

Reply:
Done!